### DISCUSSION TITLE
Cannot read property 'tagName' of null on razorpay payment in nextJs

### DISCUSSION THREAD

**Discussion by rajatgalav** [vercel/next.js#21683] [Help]
I am getting tagName of null error on razorpay integration. This error is related to head manager.

![Screenshot from 2021-01-30 07-20-01](https://user-images.githubusercontent.com/29925273/106353539-5c5b1900-6311-11eb-926b-ec4e13179c9b.png)

Link of github repository https://github.com/rajatgalav/razorpay-demo

It has a Head component where i am showing title and a link tag.
And if i remove Head component from index.js file, problem does not occur. But i want head component also in my project.

Head is imported from next/head package of NextJs

**Comment by ponikar** [comment]
This is development error! There is chance you won't get it in production!

**Reply by sohhamm** [reply]
how do we fix it in development facing the same issue right now

**Reply by ponikar** [reply]
Okay First of all I was wrong. I am getting this error in production in the console.
I did not get any solution for this.

Still, this error is not bothering me much. So I guess you can ignore it. If there is no impact on your project

**Comment by jatinderaujla** [comment]
I am facing the same issue when I hit success or failure on the Razorpay payment gateway page after that I got this issue.  If anyone has a solution for it please help to resolve the issue. I think this issue arises because the Razorpay script tries to make changes in the head tag of the HTML document and NEXTJS should not be able to make updates that why we are getting this issue.
